Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1293,10 +1293,15 @@ def get_ioarules(rule_ids: list[str]) -> dict:
         :param rule_ids: IDs of the requested ioa rule.
         :return: Response json of the get ioa rule entities endpoint (ioa rule objects)
     """
+    print("RuleID:")
+    print(rule_ids)
+
+    params = { 'ids': rule_ids }
```

Comment:
```suggestion
    params = {'ids': rule_ids}
```

Reply:
We do not use `print` in integrations :)

Reply:
Forgot to delete those haha.